jenkins/PerfCI_Chaos/04_PerfCI_Chaos_Upgrade_OpenShift_Deployment/Jenkinsfile

```diff
@@ -264,14 +258,12 @@ END
                                 } else {
                                     echo "Reached maximum retry attempts for workload ${workload}. Moving to the next workload..."
                                 }
-                            }
```

Comment:
Above here, do you need to check that the `sudo podman run` succeeds before `success = true`?

Reply:
I dont think it necessary

Reply:
Why?  It's a simple check, and things can go wrong.

Reply:
Its only on failure

Reply:
It looks like you're relying on something raising an excception if something goes wrong, but what if the `podman run` simply fails?  Shouldn't that be treated as a failure?

Reply:
If podman run will fail, I prefer that all the run the will fail.

Reply:
Right, but that's why you should check whether the podman run fails -- if it fails, then you fail the entire run.